auxx/Main.hs

```diff
@@ -70,64 +70,91 @@ correctNodeParams AuxxOptions {..} np = do
             correctNodeParams opts =<< CLI.getNodeParams loggerName cArgs nArgs
 
-        let toRealMode :: AuxxMode a -> RealMode EmptyMempoolExt a
+        let
+            toRealMode :: AuxxMode a -> RealMode EmptyMempoolExt a
             toRealMode auxxAction = do
                 realModeContext <- ask
-                let auxxContext =
-                        AuxxContext
+                let auxxContext = AuxxContext
                         { acRealModeContext = realModeContext
-                        , acTempDbUsed = tempDbUsed }
+                        , acTempDbUsed      = tempDbUsed
+                        }
                 lift $ runReaderT auxxAction auxxContext
             vssSK = fromMaybe (error "no user secret given")
                               (npUserSecret nodeParams ^. usVss)
-            sscParams = CLI.gtSscParams cArgs vssSK (npBehaviorConfig nodeParams)
+            sscParams =
+                CLI.gtSscParams cArgs vssSK (npBehaviorConfig nodeParams)
 
-        bracketNodeResources nodeParams sscParams (txpGlobalSettings pm) (initNodeDBs pm epochSlots) $ \nr -> Production $
-            let NodeContext {..} = nrContext nr
-                modifier = if aoStartMode == WithNode
-                           then runNodeWithSinglePlugin pm nr
-                           else identity
-                auxxModeAction = modifier (auxxPlugin pm opts command)
-             in runRealMode pm nr $ \diffusion ->
-                    toRealMode (auxxModeAction (hoistDiffusion realModeToAuxx toRealMode diffusion))
+        bracketNodeResources (pcBlkSecurityParam pc)
+                             nodeParams
+                             sscParams
+                             (txpGlobalSettings pm)
+                             (initNodeDBs pm pc)
+            $ \nr ->
+                  Production
+                      $ let
+                            NodeContext {..} = nrContext nr
+                            modifier         = if aoStartMode == WithNode
+                                then runNodeWithSinglePlugin pm pc nr
+                                else identity
+                            auxxModeAction =
+                                modifier (auxxPlugin pm pc opts command)
+                        in
+                            runRealMode pm pc nr $ \diffusion ->
+                                toRealMode
+                                    (auxxModeAction
+                                        (hoistDiffusion realModeToAuxx
+                                                        toRealMode
+                                                        diffusion
+                                        )
+                                    )
 
     cArgs@CLI.CommonNodeArgs {..} = aoCommonNodeArgs
     conf = CLI.configurationOptions (CLI.commonArgs cArgs)
-    nArgs =
-        CLI.NodeArgs {behaviorConfigPath = Nothing}
+    nArgs = CLI.NodeArgs {behaviorConfigPath = Nothing}
```

Comment:
Up above there was:
```
     sscParams = CLI.gtSscParams cArgs vssSK (npBehaviorConfig nodeParams)
```
which was wibbled onto two lines and here we have two lines wibbled onto one.

Both changes are just noise with zero semantic change. We should try to keep these semantically null changes out of our PRs.